packages/rest-typings/src/v1/oauthapps.ts

```diff
@@ -1,64 +1,13 @@
 import type { IOAuthApps, IUser } from '@rocket.chat/core-typings';
-import Ajv from 'ajv';
 
-const ajv = new Ajv({
-	coerceTypes: true,
-});
-
-export type OauthAppsGetParams = { clientId: string } | { appId: string };
-
-const oauthAppsGetParamsSchema = {
-	oneOf: [
-		{
-			type: 'object',
-			properties: {
-				clientId: {
-					type: 'string',
-				},
-			},
-			required: ['clientId'],
-			additionalProperties: false,
-		},
-		{
-			type: 'object',
-			properties: {
-				appId: {
-					type: 'string',
-				},
-			},
-			required: ['appId'],
-			additionalProperties: false,
-		},
-	],
-};
-
-export const isOauthAppsGetParams = ajv.compile<OauthAppsGetParams>(oauthAppsGetParamsSchema);
+import type { OauthAppsGetParams, UpdateOAuthAppParams } from '..';
 
 export type OauthAppsAddParams = {
 	name: string;
 	active: boolean;
 	redirectUri: string;
 };
 
-const OauthAppsAddParamsSchema = {
-	type: 'object',
-	properties: {
-		name: {
-			type: 'string',
-		},
-		active: {
-			type: 'boolean',
-		},
-		redirectUri: {
-			type: 'string',
-		},
-	},
-	required: ['name', 'active', 'redirectUri'],
-	additionalProperties: false,
-};
-
-export const isOauthAppsAddParams = ajv.compile<OauthAppsAddParams>(OauthAppsAddParamsSchema);
```

Comment:
Did you really mean to remove this? :thinking: 
This is causing some errors since this is still being imported in the `server/v1/oauthapps.ts` file (in order to validate the `oauth-apps.create` endpoint's params)